src/System.Security.Cryptography.Cng/tests/Configurations.props

```diff
@@ -2,6 +2,8 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
+      netfx-Windows_NT;
```

Comment:
HashData was added in net462 to allow us to build the DSACng (and DSAOpenSsl) class in a more meaningful way, along with SignData.  I thought all of those were part of netstandard20 already, in the list of members that we handwave about for net461